### DISCUSSION TITLE
How can we add Alpine plugins using @filamentScripts?

### DISCUSSION THREAD

**Discussion by Rosendito** [filamentphp/filament#10893] [Help] [answered]
Hey guys,

I recently updated filament and livewire in my project where i was using filament notifications and some plugins in my frontpage. Updating filament i needed to change the livewire scripts setup to `@filamentScripts`.

Since filament/livewire compiles alpine, how can I extend the alpine plugins?

I know we can compile ourselves changing `@livewireScripts` to `@livewireScriptsConfig` but, using filament I didn't see anywhere in the documentation how to do it.

Thanks for your support.

**Comment by Rosendito** [comment]
I found the solution in this discussion #7493 :)

**Reply by leandrocfe** [reply]
Share the code and mark as answered please

**Reply by Rosendito** [reply, marked as answer]
We need to import the plugins by cdn before the `@filamentScripts` initialization.

```blade

<!-- Alpine Plugins -->
<script defer src="https://cdn.jsdelivr.net/npm/@alpinejs/collapse@3.x.x/dist/cdn.min.js"></script>

@filamentScripts
```
